src/global.h

```diff
@@ -362,15 +362,214 @@ class CCustomEvent : public QEvent
 // command line parsing, TODO do not declare functions globally but in a class
 QString UsageArguments ( char** argv );
 
-bool GetFlagArgument ( char** argv, int& i, QString strShortOpt, QString strLongOpt );
-
-bool GetStringArgument ( int argc, char** argv, int& i, QString strShortOpt, QString strLongOpt, QString& strArg );
-
-bool GetNumericArgument ( int     argc,
-                          char**  argv,
-                          int&    i,
-                          QString strShortOpt,
-                          QString strLongOpt,
-                          double  rRangeStart,
-                          double  rRangeStop,
-                          double& rValue );
+//============================================================================
+// CMsgBoxes class:
+//  Use this static class to show basic Error, Warning and Info messageboxes
+//  For own created message boxes you should still use
+//    CMsgBoxes::MainForm() and CMsgBoxes::MainFormName()
+//============================================================================
```

Comment:
Is this comment syntax used throughout the code - except for ASIO? We should remain consistent.

Reply:
> Is this comment syntax used throughout the code - except for ASIO? We should remain consistent.

AFAIK it was already used at some places (and also in many libraries), but various other styles are used too (so not very consistent anyway)...

Personally I find this the most readable style, especially when quick scrolling through a file. But maybe that is because this was the mandatory style I had to use for many, many years, so it became a habit ;=) 
 
I can adapt if you want, but forgive me if I frequently fall back into those habits ;=)